sql/testdata/no_primary_key

```diff
@@ -38,8 +38,8 @@ ALTER TABLE t ADD c STRING
 statement ok
 INSERT INTO t VALUES (5, 6, '7')
 
-query IIT
-select * from t rowsort
+query IIT rowsort
```

Comment:
The rowsort isn't necessary because we're using `experimental_unique_int`, correct? Is there an issue filed to fix that (e.g. reverse the bits)?